metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-splunk",
-  "version": "7.3.1-rc0",
+  "version": "7.3.1",
```

Comment:
please don't adjust the version. We will do this during the next release.

Reply:
Woops! Forgot to replace the rc.